### DISCUSSION TITLE
Docs is wrong?

### DISCUSSION THREAD

**Discussion by ramvyr** [Shopify/storefront-api-feedback#250] [Help] [answered]
I want to do a page search, docs has a query query for this, but when using it I get an error.
When checking in graphiql - syntax error is highlighted
<img width="1291" alt="doc" src="https://github.com/user-attachments/assets/1a724ccc-57ee-4725-b13c-f7acffaba43b">

<img width="1336" alt="graph" src="https://github.com/user-attachments/assets/f0342908-31f4-417e-8d31-078a7a0afeac">

**Comment by vixdug** [comment]
Please post your version and your variables. Taking a very simple example here this works without any errors: 

```
query getShopDetails {
	pages(first: 5, query: "title:title") {
		edges {
			node {
				title
			}
		}
	}
}
```

**Reply by ramvyr** [reply]
> Please post your version and your variables. Taking a very simple example here this works without any errors:
> 
> ```
> query getShopDetails {
> 	pages(first: 5, query: "title:title") {
> 		edges {
> 			node {
> 				title
> 			}
> 		}
> 	}
> }
> ```

Thanks for reply.
Does it work for you? Not for me.
Version - unstable or any other
<img width="1385" alt="gql" src="https://github.com/user-attachments/assets/39e90676-5974-4161-9d12-4783e917735e">

**Reply by vixdug** [reply]
What API endpoint are you hitting? It should be `/api/unstable/graphql.json`  and not `/admin/api/graphql.json`

**Reply by ramvyr** [reply]
> What API endpoint are you hitting? It should be `/api/unstable/graphql.json` and not `/admin/api/graphql.json`

That's right, I'm using the correct entry point.
I use curl or use the built-in shopify app development utility - graphiql, the result is the same. It doesn't know what query is in the query
<img width="1422" alt="ql1" src="https://github.com/user-attachments/assets/ac3186c9-6123-4f1e-bb04-5195e0db08f1">
<img width="1422" alt="gl2" src="https://github.com/user-attachments/assets/5feb9919-eb94-4c10-bb74-07d9c11fde62">
<img width="869" alt="curl" src="https://github.com/user-attachments/assets/cef6b574-18bd-4b2a-8b85-5003ee8657cf">

**Reply by corygibbons** [reply, marked as answer]
@ramvyr You're querying the Admin API and not the Storefront API. As pointed out [above](https://github.com/Shopify/storefront-api-feedback/discussions/250#discussioncomment-10784365) your endpoint should not have `/admin` in it.

**Reply by ramvyr** [reply]
> @ramvyr You're querying the Admin API and not the Storefront API. As pointed out [above](https://github.com/Shopify/storefront-api-feedback/discussions/250#discussioncomment-10784365) your endpoint should not have `/admin` in it.

I think graphiql misled me and i lost the thread on storefront api. 
thank you all for your replies.

but once again I didn't get a result.

<img width="751" alt="store" src="https://github.com/user-attachments/assets/7c1e71da-a35b-4ec7-93d6-74112ad121e4">

**Reply by corygibbons** [reply]
@ramvyr Based on the `UNSUPPORTED_MEDIA_TYPE` error it looks like a formatting issue.

Update the values in this snippet and give it a try:

I escaped the `""` around `title:contact`.

```sh
curl -X POST "https://SHOP.myshopify.com/api/unstable/graphql.json" \
-H "X-Shopify-Storefront-Access-Token: TOKEN" \
-H "Content-Type: application/json" \
-d '{"query": "query getShopDetails { pages(first: 5, query: \"title:contact\") { edges { node { title } } } }"}'
```

**Reply by ramvyr** [reply]
Thank you, great, it works!
